setup.py

```diff
@@ -1,34 +1,37 @@
 from setuptools import setup, find_packages
 
 required_packages = [
-    'graphql-core>=2.1',
-    'graphql-server-core>=1.1.1',
-    'sanic>=0.5.1',
-    'pytest-runner'
+    "graphql-core>=2.1,<3",
+    "graphql-server-core>=1.1.1,<2",
+    "sanic>=0.5.1,<19",
 ]
 
+tests_require = ["pytest>=2.7.3", "aiohttp>=3.5.0,<4", "yarl>=1.0,<2.0", "jinja2==2.9.0"]
+
 setup(
-    name='Sanic-GraphQL',
-    version='1.1.0',
-    description='Adds GraphQL support to your Sanic application',
-    long_description=open('README.rst').read(),
-    url='https://github.com/graphql-python/sanic-graphql',
-    download_url='https://github.com/graphql-python/sanic-graphql/releases',
-    author='Sergey Porivaev',
-    author_email='porivaevs@gmail.com',
-    license='MIT',
+    name="Sanic-GraphQL",
+    version="1.1.0",
+    description="Adds GraphQL support to your Sanic application",
+    long_description=open("README.rst").read(),
```

Comment:
Would be good to provide encoding as `utf-8` in case any emoji is being used on the readme body.